doc/main/tbb_userguide/Controlling_Chunking_os.rst

```diff
@@ -4,8 +4,11 @@ Controlling Chunking
 ====================
 
 
-Chunking is controlled by a *partitioner* and a *grainsize.*\  To gain
-the most control over chunking, you specify both.
+Chunking is the process of dividing up the overall work into individual jobs,
```

Comment:
AFAIR, we do not use **job** in the documentation, maybe it is better to say chunks or sub-ranges? I would not say tasks/jobs because paritioners can proceed several chunks/ranges inside one task.

Reply:
The idea of a task is new to me - and I'm not finding it in the docs (loops section anyway). How do the overall work, tasks, and chunks mix?